src/constants/statusConstants.js

```diff
@@ -0,0 +1,14 @@
+const statusConstants = {
+  STATUS_UP: 'up',
```

Comment:
instead of `default` export here. Export each constant. 
Like this:
```js
export const STATUS_UP = 'up';
export const STATUS_DOWN: 'down',
...
```